src/pages/MoviePostWrite/WriteFirstStep.jsx

```diff
@@ -0,0 +1,158 @@
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+
+function WriteFirstStep({
+  onClickEnroll,
+  myMovies,
+  onClickMovie,
+  selectMovie,
+  history,
+}) {
+  const allMyMovies = myMovies.map(movie => (
+    <Poster
+      img={movie.poster_image}
+      onClick={() => onClickMovie(movie)}
+      myMovies={myMovies}
+      movie={movie}
+      key={movie.id}
+    >
+      <SelectPoster
+        movie={movie}
+        selectMovie={selectMovie}
+        isSelected={
+          selectMovie && selectMovie.find(select => select === movie)
+            ? true
+            : false
```

Comment:
```suggestion
          selectMovie.find(select => select === movie)
```
selectMovie의 초기값이 빈 배열이라면 이렇게만 해주어도 되지 않을까요?

Reply:
감사합니다!